templates/scenarios/csharp/command-and-response/teamsapp.local.yml.tpl

```diff
@@ -44,5 +44,5 @@ provision:
   - uses: teamsApp/update # Apply the Teams app manifest to an existing Teams app in Teams Developer Portal. Will use the app id in manifest file to determine which Teams app to update.
     with:
       appPackagePath: ./build/appPackage/appPackage.${{TEAMSFX_ENV}}.zip # Relative path to this file. This is the path for built zip file.
-    writeToEnvironmentFile: # Write the information of installed dependencies into environment file for the specified environment variable(s).
+    writeToEnvironmentFile: # Write the information of created resources into environment file for the specified environment variable(s).
```

Comment:
Will ttk create a new Teams app in `teamsApp/update`  action, or just update it?

Reply:
Thanks, as discussed with PM,  will remove output for teamsApp/update action.